.eslintrc

```diff
@@ -6,6 +6,9 @@
     //     "eslint:recommended",
     //     "plugin:react/recommended",
     // ],
+    "extends": [
+        "eslint:recommended",
+    ],
```

Comment:
I am completely on the fence about the `comma-dangle`. I lean toward allowing it because of the benefit of clarity in _diffs_ (you don't see the "previous" line and the new line, just what is added).